models/instrument.py

```diff
@@ -1588,21 +1653,15 @@ def __init__(self, **kwargs):
 
     @classmethod
     def get_section_ids(cls):
-
-        """
-        Get a list of SensorSection identifiers for this instrument.
-        """
-        return [0]
+        return [ '0' ]
```

Comment:
We're not consistent about this, and probably should be.  When the documentation of a method that overrides a method in a parent class is identical to the parent class' docstring, should it be in both places?  Argument for: good to have documentation readily available.  Argument against: harder to maintain, because if we want to edit it, we have to find and edit it in lots of places.

Ideally, I'd like Python's "help" and documentation generation functionalty to work in the mode of "if this method doesn't have documentation, but the same method in a parent class does, show the parent class' documentation", but I don't think that it does.  So, it's up to the documentation reader to decide to do that manually.

Reply:
I can think of many cases where the doc for a subclass method would be similar but add some details. Lets say the bare minimum is to have at least a short (one line) docstring on base and subclass methods just so people don't have to go looking around too much. But if there are complicated explanations we should just say "see the base class docstring" or something like that to prevent duplication.